RadixWallet/Features/AccountHistory/TransactionHistory+View.swift

```diff
@@ -0,0 +1,611 @@
+import ComposableArchitecture
+import SwiftUI
+
+extension TransactionHistory.State {
+	var showEmptyState: Bool {
+		sections.isEmpty && !loading.isLoading
+	}
+}
+
+// MARK: - TransactionHistory.View
+extension TransactionHistory {
+	@MainActor
+	public struct View: SwiftUI.View {
+		private let store: StoreOf<TransactionHistory>
+
+		public init(store: StoreOf<TransactionHistory>) {
+			self.store = store
+		}
+
+		public var body: some SwiftUI.View {
+			NavigationStack {
+				WithViewStore(store, observe: { $0 }, send: { .view($0) }) { viewStore in
+					let accountHeader = AccountHeaderView(account: viewStore.account)
+
+					let selection = viewStore.binding(get: \.currentMonth, send: ViewAction.selectedMonth)
+
+					VStack(spacing: .zero) {
+						VStack(spacing: .small2) {
+							HScrollBarDummy()
+
+							if !viewStore.activeFilters.isEmpty {
+								ActiveFiltersView.Dummy()
+							}
+						}
+						.padding(.top, .small2)
+						.padding(.bottom, .small1)
```

Comment:
What is the functionality of this? I couldn't figure just from the code :(

Reply:
Any time there is a dummy, the purpose is to get the size of some view whose size would be affected by dynamic font sizes. Either by adding it as a spacer, or by reading it's size using preferences, where it is impractical or impossible to read the size of the actual thing.